src/Blockcore.Indexer.Cirrus/Controllers/CirrusQueryController.cs

```diff
@@ -99,6 +100,23 @@ public async Task<IActionResult> GetStandardTokenContractByAddress([MinLength(30
          return Ok(contract);
       }
 
+      [HttpGet]
+      [Route("contract/StandardToken/{address}/{filterAddress}")]
+      [SlowRequestsFilteerAttribute]
```

Comment:
@DavidGershony will this be an issue for the filter because of the additional param?

Reply:
So as discussed this not such a big deal as it will at worst result in two updates to disk on computation of the standard token table

Reply:
make the Route all lower-case characters.